MovieNight/GenreViewController.swift

```diff
@@ -24,26 +30,36 @@ class GenreViewController: UIViewController , UITableViewDelegate, UITableViewDa
     }
     
     func loadData() {
-        fetchForGenres()
+        fetchForGenres(with: page)
```

Comment:
As you renamed the ApiClient methods, all these `fetchForGenres()` in the ViewControllers can be renamed as well.